geopy/distance.py

```diff
@@ -122,12 +123,17 @@
 
 def lonlat(x, y, z=0):
     """
-    Helper function that can be used for more explicit and obvious calling of the distance methods
+    distance accepts coordinates in (y, x)/(lat, lon) order, while some other libraries/systems might use 
+    (x, y)/(lon, lat).
+    This helper function introduced here for solving this problem. It accepts coordinates in a form of (x, y)/(lon, lat)
+    and returns Point that is safe to use with distance
```

Comment:
1. A sentence should end with a dot (or some other relevant punctuation mark).
2. It seems to me that these two sentences are intended to be in their own paragraph. Paragraphs should be separated with one empty line, which is missing there.
3. These two sentences don't sound natural to me. Let me propose the following phrasing:

        This function provides a convenient way to convert coordinates of 
        the ``(x, y)/(lon, lat)`` format to a :class:`geopy.point.Point` instance.

        Example::